routes/create_user.routes.js

```diff
@@ -0,0 +1,16 @@
+module.exports = function(app, db) {
+
+    app.post('/create_user', (req, res) => {
+        let user_session_cookie = Math.floor(Math.random()*Number.MAX_SAFE_INTEGER);
```

Comment:
Should we check if the number isn't (by a slim chance) already used by another user? I don't think that we'll run into this problem, just something to consider.

Reply:
bei größeren Anwendungen sollte man das vielleicht machen, bei uns ist das aber egal, denke ich. Das ist so eine große Zahl, selbst bei vielen usern ist die Chance, dass eine doppelt ist, praktisch nicht existent.